Configurations/50-win-onecore.conf

```diff
@@ -61,4 +61,43 @@ my %targets = (
         ex_libs         => "onecore.lib",
         multilib        => "-arm64",
     },
+
+	# Universal Windows Platform (UWP) App Support
+
+    "VC-WIN32-UWP" => {
+        inherit_from    => [ "VC-WIN32-ONECORE" ],
+        lflags          => add("/APPCONTAINER"),
+        defines         => add("WINAPI_FAMILY=WINAPI_FAMILY_APP",
+                               "_WIN32_WINNT=0x0A00"),
+        dso_scheme      => "",
+        disable         => [ 'ui-console', 'stdio', 'async' ],
```

Comment:
Can't we just disable "uplink" here for all targets? Then you don't need to specify it at configure time?

Reply:
You are right. I have modified that and test compiling. Please wait.

Reply:
I have modified via your advice. But I found compiling failed because the "no-uplink" is not applied. I want to find you for help.

Reply:
I have modified via your advice. But I found compiling failed because the "no-uplink" is not applied. I want to find you for help. @mattcaswell

Reply:
@levitte is probably the best person to help here.

Reply:
There's a catch, the use in `inherit_from` attributes happens before this attribute is looked at. This is a fragility in Configure...

This can be remedied with late evaluation (I have an experimental PR that I didn't finish), maybe now's the time to introduce it....

Reply:
@levitte Thank you. Got it. But what shoud I do for fixing the problem? Do I need to modify my PR? If needed, I wonder to know what I should modify in this PR?

Reply:
No, there's nothing you can do about it here...  or well, you *could* add a comment along the lines of this, before the first UWP target:

``` perl
# TODO
# The 'disable' attribute should have 'uplink' and 'asm' as well...  however, these are checked in
# some 'inherit_from', which is processed very early, before the 'disable' attributes are seen.
# This is a problem that needs to be resolved in Configure first.
```

Reply:
@levitte Thank you. I have added the TODO note. So can I click the "Resolve conversation" button?